db/migrate/20150522121048_add_command_type_to_stages.rb

```diff
@@ -0,0 +1,7 @@
+class AddCommandTypeToStages < ActiveRecord::Migration
+  def change
+    change_table :stages do |t|
+      t.string :command_type
```

Comment:
How about set a `null: false, default: 'shell_script'` or whatever the correct value would be?